content: add description and title metas (#1600)
nuxt/nuxtjs.org: content/en/team.md

---
template: page
title: 'NuxtJS teams'

> description: 'The NuxtJS and its ecosystem development is lead by an international team. Our committed team works every day to bring NuxtJS forward.'

Reviewer: ```suggestion
description: 'Nuxt development is lead by an international team. Our committed team works every day to move Nuxt forward.'
```
